masterlist.yaml

```
@@ -23530,7 +23639,7 @@ plugins:
         condition: 'active("mihailbeholder.esp") and checksum("mihailbeholder.esp", 447993FF)'
     msg:
       - <<: *compatNotes
-        subs: [ 'https://www.nexusmods.com/skyrimspecialedition/articles/2761']
+        subs: [ 'https://www.nexusmods.com/skyrimspecialedition/articles/2761/' ]
```

This should be in a separate PR as it's unrelated to the common theme & not a plugin otherwise edited in a way consistent with that theme:

```suggestion
        subs: [ 'https://www.nexusmods.com/skyrimspecialedition/articles/2761']
```